components/hab/src/main.rs

```diff
@@ -1440,18 +1438,18 @@ fn enable_features_from_env(ui: &mut UI) {
                 "  * {:?}: HAB_FEAT_{}={}",
                 feature.0,
                 feature.1,
-                henv::var(format!("HAB_FEAT_{}", feature.1)).unwrap_or("".to_string())
+                henv::var(format!("HAB_FEAT_{}", feature.1)).unwrap_or_default()
             ))
             .unwrap();
         }
     }
 }
 
 fn handle_ctl_reply(reply: SrvMessage) -> result::Result<(), SrvClientError> {
-    let mut bar = pbr::ProgressBar::<io::Stdout>::new(0);
-    bar.set_units(pbr::Units::Bytes);
-    bar.show_tick = true;
-    bar.message("    ");
+    let mut progress_bar = pbr::ProgressBar::<io::Stdout>::new(0);
```

Comment:
Clippy complains about the name `bar`.

Reply:
I guess to prevent you from adding lots of meaningless variables named `foo`, `bar`, `baz`, etc.?